libs/blocks/fragment/fragment.js

```diff
@@ -117,15 +113,9 @@ export default async function init(a) {
   }
 
   updateFragMap(fragment, a, relHref);
-
-  if (a.dataset.manifestId) {
-    if (inline) {
-      setManifestIdOnChildren(sections, a.dataset.manifestId);
-    } else {
-      fragment.dataset.manifestId = a.dataset.manifestId;
-    }
+  if (a.dataset.manifestId || a.dataset.adobeTargetTestid) {
+    updateFragDataProps(a, inline, sections, fragment);
```

Comment:
we should dynamically import personalization.js and use updateFragDataProps only if needed
```suggestion
     const { updateFragDataProps } = await import('../../features/personalization/personalization.js');
     updateFragDataProps(a, inline, sections, fragment);
```

Reply:
pushed fix for this. Thanks.